types/types.go

```diff
@@ -180,8 +180,8 @@ type ImageDestination interface {
 	// FIXME? This should also receive a MIME type if known, to differentiate between schema versions.
 	// If the destination is in principle available, refuses this manifest type (e.g. it does not recognize the schema),
 	// but may accept a different manifest type, the returned error must be an ManifestTypeRejectedError.
-	PutManifest(manifest []byte) error
-	PutSignatures(signatures [][]byte) error
+	PutManifest(manifest []byte, instanceDigest *digest.Digest) error
+	PutSignatures(signatures [][]byte, instanceDigest *digest.Digest) error
```

Comment:
All three updated methods should document `instanceDigest` (and, ideally, this should be propagated all over the copy&pasted comments; that part is non-blocking for me).

Reply:
Okay, added.